public/manifest.json

```diff
@@ -4,7 +4,7 @@
   "icons": [
```

Comment:
as you deleted usage manidest.json you should have deleted this file as its no longer required